malaria-ios/ViewControllers/DailyStatsTableViewController.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import UIKit
+
+class DailyStatsTableViewController : UITableViewController{
+
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 3
+    }
+    
+    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
+        return false
+    }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return 125.0
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let row = indexPath.row
+        let section = indexPath.section
+        
+        if row == 0 {
+            let cell = tableView.dequeueReusableCellWithIdentifier("medicineLastTakenCell") as! UITableViewCell
+            return cell
+        }else if row == 1 {
+            let cell = tableView.dequeueReusableCellWithIdentifier("dosesCell") as! UITableViewCell
```

Comment:
Instead of using magic numbers here to check you can use enums to give them meaning such as the following for clarity.  The last count one would also give you the proper value for numberOfRowsInSection so it's a simpler change if you have to add more rows or change their order.

`DailyStatsMedicineLastTakenCell`
`DailyStatsDosesCell`
`DailyStatsCellCount`